modules/gapi/CMakeLists.txt

```diff
@@ -207,6 +207,10 @@ if(OPENCV_GAPI_INF_ENGINE)
   ocv_target_link_libraries(${the_module} PRIVATE ${INF_ENGINE_TARGET})
 endif()
 
+if (HAVE_NGRAPH)
+    ocv_target_link_libraries(${the_module} PRIVATE ngraph::ngraph)
```

Comment:
@alalek I noticed that `InferenceEngine` and `ngraph` are handled different way, do we need to make alias to `ngraph::ngraph` like `NGRAPH_TARGET`?

Reply:
Does it make sense not to use `InferenceEngine` separately from `ngraph` by default?

Reply:
> separately

`find_package(InferenceEngine)` and `find_package(ngraph)`.

Check DNN module integration.

Reply:
I believe it's so similar to my implementation, isn't it? Except maybe this: https://github.com/opencv/opencv/blob/master/modules/dnn/CMakeLists.txt#L162 in DNN definitions set globally, in G-API we need this only for `opencv_test_gapi`, so far

Reply:
Is this question closed now?

Reply:
@alalek Is it OK?